Makefile

```diff
@@ -6,6 +6,15 @@ target/http-api-gateway-%.jar: $(SOURCES) $(RESOURCES)
 
 uberjar: target/http-api-gateway-%.jar
 
+docker-build:
+	docker buildx build --platform linux/amd64,linux/arm64 -t fluree/http-api-gateway:latest -t fluree/http-api-gateway:$(shell git rev-parse HEAD) --build-arg="PROFILE=prod" .
```

Comment:
This might need a `--load` arg to actually add the image to the local Docker environment. Or did you find that it wasn't necessary (I've been hoping they would default to that in a future release, so maybe they did)?

Reply:
@cap10morgan Ya... unfortunately `--load` can be used with multi-platform builds. I could keep the multi-platform arg in the `docker-push` target and remove it from `docker-build`? I agree though, right now running `docker-build` just caches the artifact and is largely useless. What do you think?

Reply:
Ohhh that's right. I _think_ I just removed the multiple architectures and added `--load` for the `build` target in other repos (but kept it in `push` as you propose). That seems the most useful / intuitive to me, at least.